### PR TITLE
chore: update badges for the repository

### PR DESCRIPTION
**Pull request by ovflowd**
This PR simply updates the badges, including a "Powered by Vercel" badge and removing the Twitter badge that is a little bit useless. (We have many socials, doesn't make sense to share just one)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
+    <img src="https://img.shields.io/badge/license-MIT-blue" />
   </a>
   <a title="Localised" href="https://crowdin.com/project/nodejs-website">
-    <img src="https://badges.crowdin.net/nodejs-website/localized.svg">
+    <img src="https://badges.crowdin.net/nodejs-website/localized.svg" />
   </a>
-  <a title="Follow on Twitter" href="https://twitter.com/Nodejs">
-    <img src="https://img.shields.io/twitter/follow/Nodejs.svg?style=social&label=Follow%20@Nodejs">
+  <a title="Vercel" href="https://vercel.com">
+    <img src="https://img.shields.io/badge/powered%20by-Vercel%20%E2%96%B2-black" />
```
Comment (AugustinMauroy) [inline README.md]: That doesn't look good in dark mode.
Should we use GMF theme detector ?

Capture d’écran 2023-06-29 à 10 47 13

Comment (ovflowd) [reply README.md]: I am in the Dark Mode and it looks nice, but sure, we can have two different images. One for dark mode and one for light mode. Let me try something.

**Comment by tniessen** [comment]
> removing the Twitter badge that is a little bit useless

On the other hand, are any of them useful? :)

**Review by AugustinMauroy — APPROVED** [review]
LGTM

**Comment by ovflowd** [comment]
> On the other hand, are any of them useful? :)

You know you have a point right? But let's pretend these badges have any value lol

**Comment by tniessen** [comment]
Since this is just the website repo, I won't push in either direction, but FWIW, I don't think we should use it as a Vercel ad space.

**Comment by ovflowd** [comment]
> Since this is just the website repo, I won't push in either direction, but FWIW, I don't think we should use it as a Vercel ad space.

What would be the correct space? nodejs/build has a list of sponsors for the build infra. Shouldn't the website also show a badge of one of the sponsors?

I'm not sure what is the usual place for such kind of badges/"advertisement", so if there's a TSC-approved way of doing things, let me know :)

**Comment by bmuenzenmeyer** [comment]
I've done extensive usability research into README badges. In many cases they are not accessible to screen readers. Apple Voiceover, Windows, even JAWS will sometimes read verbatim data url content. 

They look nice, yes, but I agree with @tniessen they are sorta like bumperstickers

**Comment by ovflowd** [comment]
> They look nice, yes, but I agree with @tniessen they are sorta like bumperstickers

That part I get. But I still would like to know from @tniessen what is the correct approach (or TSC-approved approach) of supporting our sponsors.

**Comment by tniessen** [comment]
@ovflowd I really don't have enough of an opinion to care about the website repo's README, nor am I speaking on behalf of the TSC. IIRC, Vercel employees already claimed that Node.js would be switching to Vercel before it was decided. If you think this is the way of "supporting our sponsors", go for it.

**Comment by ovflowd** [comment]
Sorry @tniessen my intent here isn’t at all to make you uncomfortable either to abide by things that are questionable at any given point.

I’m genuinely more than Ok with closing this PR and opening an issue on nodejs/admin and seeking guidance for what’s the best approach here 🙏

**Comment by Harkunwar** [comment]
I don't have much strong opinions on this either. If they're sponsoring us with hosting the website more than just the free tier, I think I'm okay with putting them in the README, unless of course anyone else has a better idea.

**Comment by ovflowd** [comment]
cc @richardlau as the build repository is an excellent example of showcasing sponsors' badges. Do you have any opinion on this? (the whole argumentation regarding we adding this or not, based on Tobias's points)

**Comment by richardlau** [comment]
> cc @richardlau as the build repository is an excellent example of showcasing sponsors' badges. Do you have any opinion on this? (the whole argumentation regarding we adding this or not, based on Tobias's points)

I have no opinion on adding things to the README for this (the website) repository.

The stuff in the Build WG's README predates my involvement. In general I think it does make sense to acknowledge the generous donations of infrastructure resources that keep the Node.js project going. If we do end up hosting the website on Vercel we should add them there alongside our other providers who have donated resources to Node.js infrastructure.

**Comment by ovflowd** [comment]
> g. If we do end up hosting the website on Vercel we should add them there alongside our other providers who have donated resources to Node.js infrastructure.

Gotcha. From what I'm getting, the general "tone" here is to add a badge if/once a shift to Vercel happens on the production Website. I'm fine by keeping this open.

Also, right now DigitalOcean and CloudFlare are the sponsors for the Node.js Website Infrastructure, so I wonder if we should regardless have a "Sponsors" section below with the organisations that have donated resources to the Node.js project for keeping the Website alive?

**Comment by richardlau** [comment]
> Also, right now DigitalOcean and CloudFlare are the sponsors for the Node.js Website Infrastructure, so I wonder if we should regardless have a "Sponsors" section below with the organisations that have donated resources to the Node.js project for keeping the Website alive?

Again, I have no opinion on adding stuff to the website repository but the Build WG README sort of covers that https://github.com/nodejs/build#infrastructure-providers (the text for DigitalOcean and Cloudflare mentions nodejs.org).

**Comment by ovflowd** [comment]
cc @nodejs/website if anyone is interested on approving this PR.

**Review by mikeesto — APPROVED** [review]
I think it makes sense to acknowledge the contribution that Vercel is making by way of infrastructure beyond their free tier. A small badge on the README is far less intrusive than, for example, an ad space on the website itself. I'm in support of adding this.

**Review by bmuenzenmeyer — APPROVED** [review]
I like it.

Assuming the logo rendering problem is a quirk of the mobile app rendering.

<details><summary>mobile screenshot</summary>

</details>

**Pull request merged**